software/24.Git_GitHub/Git/README.md

```diff
@@ -0,0 +1,232 @@
+# Part 1: Git
```

Comment:
You need to add a small bulleted list here to give the main competencies that the workshop provides globally.